utils/docker-compose/network-config/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
Might be worth mentioning in the header that it depends on the FPC_CONFIG, PEER_CMD and COMPOSE_PROJECT_NAME env variables?  
Along these lines, it would also be nice if the defaults of these variables from start.sh are also set as defaults in this yaml file, so i can use this yaml file directly to  run docker-compose -f <this-file> with commands such as ps, log, etc ..  
(Not really a big deal, i figured it out quickly as docker-compose gave meaningful warnings and then figuring out necessary values by greping -r for these VARs, but still would be handy if i wouldn't have to define these vars myself :-)

Reply:
I don't know how to provide default values for the docker-compose.yml itself if one wants to use `docker-compose -f` directly. For now I will put some comments at the top of the file outlining the environment variables needed.

I added `set +x` to the `start.sh` so we should see the export statements printed out now

Reply:
while i had the "luxury version" of a single line 'echo export A="${A}" B="${B}" ...' in mind, set -x also does the job :-)

Reply:
ahhh, that's not hard to add :)